lib/lower/convert-expr.cc

```diff
@@ -555,7 +577,35 @@ class ExprLowering {
   M::Value gen(Ev::ComplexPart const &) { TODO(); }
   M::Value gendef(Ev::ComplexPart const &cp) { return gen(cp); }
   M::Value genval(Ev::ComplexPart const &) { TODO(); }
-  M::Value gen(const Ev::Substring &) { TODO(); }
+
+  M::Value gen(const Ev::Substring &s) {
+    // Get base address
+    auto baseAddr{std::visit(
+        Co::visitors{
+            [&](const Ev::DataRef &x) { return gen(x); },
+            [&](const Ev::StaticDataObject::Pointer &) -> M::Value { TODO(); },
```

Comment:
Would this just be a LoadOp?

Reply:
For the `Ev::StaticDataObject::Pointer` ?  I guess you are right, I did not dwell to much into what is in an `Ev::StaticDataObject::Pointer` to be sure what to do.

Reply:
Ok. Seems like it ought to be a load, but that's a guess.